torchmultimodal/models/flava/flava_image_encoder.py

```diff
@@ -21,9 +20,7 @@
 from torchmultimodal.modules.losses.flava import Pooler
 
 
-def to_2tuple(x):
-    if isinstance(x, collections.abc.Iterable):
-        return x
+def to_2tuple(x: int) -> Tuple[int, int]:
```

Comment:
do we even need this function? it's used only twice and can be easily replaced with `image_size = (image_size, image_size)`

Reply:
yes get rid of it in another pr